ibm/mas_devops/roles/suite_app_config/vars/predict.yml

```diff
@@ -11,6 +11,7 @@ mas_app_settings_wml_url: "{{ lookup('env', 'WML_URL') | default('https://intern
 mas_app_settings_wml_version: "{{ lookup('env', 'WML_VERSION') }}"
 mas_app_settings_predict_deployment_size: "{{ lookup('env', 'PREDICT_DEPLOYMENT_SIZE') | default('small', true) }}"
 
+mas_app_settings_watsonstudio_project_id: "{{ lookup('env', 'WS_PROJECT_ID') | default('', true) }}"
```

Comment:
Env var lookups should all be in defaults, not vars .. but seeing as there are existing problems in this role in this area will not reject this ... but next update to this role we need to sort out what should be in vars and what should be in defaults.

(The reason is that env var lookups in vars files will override any parameters passed in on command line, or via a playbook .. making env vars the **only** way to set the variable)